src/CMakeLists.txt

```diff
@@ -33,7 +34,10 @@ target_compile_definitions(mc_rbdyn_urdf PRIVATE "-DMCRBDYNURDF_BUILDING")
 set(INSTALL_PATH include/mc_rbdyn_urdf)
 install(FILES ${HEADERS} DESTINATION ${INSTALL_PATH})
 install(TARGETS mc_rbdyn_urdf
-  RUNTIME DESTINATION bin
-  LIBRARY DESTINATION lib
-  ARCHIVE DESTINATION lib
+EXPORT "${TARGETS_EXPORT_NAME}"
+LIBRARY DESTINATION "lib"
+ARCHIVE DESTINATION "lib"
+RUNTIME DESTINATION "bin"
+INCLUDES DESTINATION "${INCLUDE_INSTALL_DIR}"
```

Comment:
Please keep the indentation that was in the original file

Reply:
Is the indentation style four spaces or two spaces?  In [RBDy/src/CMakelist.txt](https://github.com/jrl-umi3218/RBDyn/blob/master/src/CMakeLists.txt), it's four spaces but in this file it's two spaces.

I changed it back to two spaces.